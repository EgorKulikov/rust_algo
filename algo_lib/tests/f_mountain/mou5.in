8192
I 5436 6475 4064
I 414 1430 4110
I 2467 3495 4086
I 6025 7033 4126
I 5902 6942 4063
I 4176 5220 4055
I 2297 3324 4088
I 6732 7765 4076
I 2982 4006 4094
I 6055 7074 4104
I 226 1256 4082
I 6307 7313 4131
I 2968 3974 4131
I 2733 3746 4116
I 4020 5035 4112
I 5611 6627 4110
I 923 1968 4053
I 5994 7009 4112
I 5670 6674 4135
I 4575 5604 4084
I 3894 4911 4108
I 2969 4014 4053
I 7071 8071 4143
I 1242 2257 4112
I 6806 7831 4092
I 5336 6349 4116
I 3166 4205 4064
I 2852 3869 4108
I 515 1517 3616
I 5507 6554 4049
I 4897 5945 4047
I 4679 5721 4059
I 3460 4474 4114
I 1125 2151 4090
I 2979 4005 4090
I 5941 6970 4084
I 983 2003 4102
I 1309 2312 4137
I 4756 5793 4068
I 256 1276 4102
I 1634 2658 4094
I 2060 3078 4106
I 1019 2020 4141
I 2086 3091 4133
I 6327 7359 4078
I 4165 5207 4059
I 2841 3888 4049
I 1293 2321 4086
I 4699 5707 4126
I 1027 2028 3618
I 4160 5192 4078
I 6949 7975 4090
I 5630 6652 4098
I 4810 5844 4074
I 5375 6417 4561
I 5164 6212 4047
I 1210 2258 4047
I 3137 4142 4133
I 5674 6714 4063
I 274 1299 4092
I 5612 6615 4137
I 623 1661 4066
I 4192 5209 4108
I 5564 6607 4057
I 4191 4542 3537
Q 28282896
I 4378 4728 3542
Q 18675445
I 7586 7753 5169
Q 30888088
I 7238 7280 2048
Q 31611170
I 929 1204 3948
Q 18303778
I 5433 5599 2048
Q 25934917
I 2047 2052 89429
Q 31270304
I 4389 4451 2048
Q 21472797
I 7888 8035 5590
Q 31632038
I 1300 1600 3790
Q 28601105
I 8137 8166 2048
Q 32088030
I 4415 4579 2048
Q 21389487
I 4903 5148 4179
Q 22476367
I 1574 1663 2048
Q 14784148
I 7649 7831 4913
Q 31239585
I 428 699 3976
Q 24274314
I 5899 6082 2048
Q 23759818
I 5090 5134 13699
Q 20928633
I 5079 5376 5567
Q 25334845
I 6995 7226 4308
Q 28382557
I 5430 5773 3572
Q 28126617
I 2124 2124 2048
Q 16605737
I 2363 2627 4026
Q 17358382
I 1383 1626 4197
Q 29829969
I 6729 6927 4683
Q 28986785
I 314 609 3819
Q 8073770
I 7856 7900 2048
Q 31797983
I 8013 8068 2048
Q 31858575
I 2508 2550 2048
Q 11392904
I 7771 7916 2048
Q 31043070
I 3836 4087 4129
Q 24564706
I 4646 4666 2048
Q 30651820
I 5097 5222 6209
Q 26928007
I 6951 7044 2048
Q 31149127
I 6201 6354 2048
Q 30188480
I 6157 6322 2048
Q 25411471
I 2549 2572 23893
Q 28283158
I 698 852 5431
Q 26677074
I 761 814 11757
Q 6400260
I 2159 2338 4961
Q 19362381
I 3375 3424 2048
Q 27561819
I 2278 2379 7188
Q 13964654
I 3176 3182 2048
Q 23205945
I 632 658 2048
Q 12644268
I 3505 3647 5714
Q 16549774
I 264 351 2048
Q 8092483
I 1856 1986 2048
Q 10254384
I 6298 6311 2048
Q 32013202
I 7072 7172 7239
Q 29703169
I 6976 7081 2048
Q 28720936
I 6766 6835 2048
Q 31990834
I 4935 4954 2048
Q 25055911
I 6754 6760 2048
Q 28558654
I 4267 4306 2048
Q 23876555
I 3134 3183 2048
Q 25985626
I 1979 1992 2048
Q 22615245
I 6795 6822 2048
Q 30984623
I 6966 6991 2048
Q 30554613
I 7329 7351 2048
Q 29665897
I 2103 2115 2048
Q 14144785
I 294 309 2048
Q 5730625
I 5781 5783 2048
Q 25068799
I 5405 5411 2048
Q 27617659
I 5666 5666 2048
Q 29285333
E
