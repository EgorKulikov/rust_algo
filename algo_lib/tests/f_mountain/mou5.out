7197
4862
7815
8013
4779
6737
7894
5472
7974
7329
8160
5566
5846
3788
7974
6339
6303
5326
6583
7339
7214
4245
4522
7606
7376
2130
8008
8031
2967
7938
6348
7802
6892
7985
7786
6642
7236
6776
1706
4764
6880
3240
5553
2929
3852
2053
2551
8026
7385
7162
8016
6245
7174
5833
6565
5517
7799
7694
7514
3460
1579
6345
7049
7433
